src/lib.rs

```diff
@@ -97,14 +97,14 @@ impl UsbBus {
     }
 
     fn endpoint_byte_count(&self, cs: &CriticalSection) -> u16 {
```

Comment:
For example (do this for all the function items accepting a `CriticalSection`):

```suggestion
    fn endpoint_byte_count(&self, cs: CriticalSection) -> u16 {
```

And then revert all the changes that add `*` and `&`.

Reply:
Will do :) thx for the suggestion